components/monitoring/prometheus/base/install-prometheus.yaml

```diff
@@ -20,3 +20,4 @@ spec:
   name: prometheus
   source: community-operators
   sourceNamespace: openshift-marketplace
+  startingCSV: prometheusoperator.0.56.3
```

Comment:
should not be there, on new installation it will install this version and will be doing updates, so it's better to not having it here -> so newest is just installed.

Reply:
The fix is to just remove the line, @Michkov ?

Reply:
yes

Reply:
removed in e278564